pkg/api/v1/types.go

```diff
@@ -869,6 +869,8 @@ type VolumeMount struct {
 	// Path within the container at which the volume should be mounted.  Must
 	// not contain ':'.
 	MountPath string `json:"mountPath"`
+	// Optional: Defaults to false (in a private propagation mode)
```

Comment:
Consider supplying a more descriptive comment, e.g.

// Use 'shared' mount propagation if true, private otherwise.  Defaults to false.